### PR TITLE
Add guide for saving .chor files

### PR DESCRIPTION
**Pull request by Daniel1464**
Now, users must save their .chor files in deploy/choreo; this guide makes sure to tell them that

### CODE REVIEW

File: docs/document-settings/saving-trajectories.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+1. Create a "choreo" folder within the "deploy" folder of your robot project. 
+The deploy folder can be found within "src/main".
+2. Open the choreo main menu and click "save project". 
+3. Navigate to your robot project's folder, then find(and select) the "choreo" folder
+you just created.
+4. Name your .chor file whatever you want.
+
+!!! warning
+    Your .chor file must be placed within the deploy/choreo folder of your robot project.
+    Not doing so will result in ChoreoLib(and PathPlannerLib) being unable to parse your project.
```
Comment (katzuv) [inline docs/document-settings/saving-trajectories.md:24]:
```suggestion
    Not doing so will result in ChoreoLib (and PathPlannerLib) being unable to parse your project.
```

File: docs/document-settings/saving-trajectories.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Saving your .chor file
+
+Once trajectories are created by choreo, they must be saved within your robot project.
```
Comment (katzuv) [inline docs/document-settings/saving-trajectories.md:3]:
```suggestion
Once trajectories are created by Choreo, they must be saved within your robot project.
```

File: docs/document-settings/saving-trajectories.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+Choreo has 2 different kinds of files: 
+a .chor file which stores general configs for your project,
+and multiple .traj files which describe individually generated trajectories.
+
+Once a .chor file is saved to a directory(folder), saving your project will
```
Comment (katzuv) [inline docs/document-settings/saving-trajectories.md:9]:
```suggestion
Once a .chor file is saved to a directory (folder), saving your project will
```

**Comment by oh-yes-0-fps** [comment]
telling them to name their `.chor` file `Choreo.chor` is silly, do `{project_name}.chor` or smthn

**Comment by Daniel1464** [comment]
> This is not accurate for Python projects

where does choreolib/pplib read for those

**Comment by shueja** [comment]
> where does choreolib/pplib read for those

Currently it reads from project root. There was discussion today about making that user-settable.

File: docs/usage/saving.md [hunk 4]
```diff
@@ -1,23 +1,51 @@
...
+
+Path following libraries, such as ChoreoLib and PathPlannerLib, require your .traj files to be located
+in a specific directory so that they can be read. For Java and C++ teams, .traj files must be placed under
+the deploy/choreo directory; for python teams, they must be placed in the root directory.
+
+### For Java and C++ teams:
+
+1. Create a "choreo" folder within the "deploy" folder of your robot project.
+   The deploy folder can be found under "src/main".
+2. Open the choreo sidebar and click "save project"(see "Sidebar Navigation" for more details).
```
Comment (shueja) [inline docs/usage/saving.md:19]:
```suggestion
2. Open the Choreo sidebar and click "Save Project"(see "Sidebar Navigation" for more details).
```

File: docs/usage/saving.md [hunk 5]
```diff
@@ -1,23 +1,51 @@
...
+3. Navigate to the location of your robot project and find the "src/main/deploy/choreo" folder within it.
+4. Name your .chor file whatever you want.
+
+### For Python teams:
+
+1. Open the choreo sidebar and click "save project"(see "Sidebar Navigation" for more details).
```
Comment (shueja) [inline docs/usage/saving.md:25]:
```suggestion
1. Open the Choreo sidebar and click "Save Project"(see "Sidebar Navigation" for more details).
```

File: docs/usage/saving.md [hunk 6]
```diff
@@ -1,23 +1,51 @@
...
 ## Open File
 
-This opens the system’s file select dialog to select the robot’s `.chor` file. This should already be in your robot project root folder.
+This opens the system’s file select dialog to select the robot’s `.chor` file. This .chor file should be under the folders "src/main/deploy/choreo"
+in your robot project.
 
 ## Save File
 
-This opens the system’s file save dialog to select where to save the robot’s `.chor` file. Choreo strongly recommends saving this file in your robot project root folder.
+This opens the system’s file save dialog to select where to save the robot’s `.chor` file. Choreo strongly recommends saving this file in your deploy/choreo folder within your project.
```
Comment (shueja) [inline docs/usage/saving.md:48]:
```suggestion
This opens the system’s file save dialog to select where to save the robot’s `.chor` file. Choreo strongly recommends saving this file in your `deploy/choreo` folder within your project.
```

File: docs/usage/saving.md [hunk 7]
```diff
@@ -1,23 +1,51 @@
...
 !!! note
-    Choreo’s file saving mechanism ties into a WPILib robot root directory, wherein a single `.chor` file lives in the project’s root folder.
+Choreo’s file saving mechanism ties into a WPILib robot root directory, wherein a single `.chor` file lives in the project’s root folder.
 
 In the sidebar, there are five actions related to saving your project, along with your project's current save location.
 
 ## Open File
 
-This opens the system’s file select dialog to select the robot’s `.chor` file. This should already be in your robot project root folder.
+This opens the system’s file select dialog to select the robot’s `.chor` file. This .chor file should be under the folders "src/main/deploy/choreo"
```
Comment (shueja) [inline docs/usage/saving.md:43]:
```suggestion
This opens the system’s file select dialog to select the robot’s `.chor` file. This .chor file should be under the folders `src/main/deploy/choreo`
```

File: docs/usage/saving.md [hunk 8]
```diff
@@ -1,23 +1,51 @@
...
+
+## Sidebar Navigation
+
 To access the sidebar, click the hamburger icon in the upper left corner.
 
 ![Sidebar image](../media/sidebar.png)
 
 General Info:
 
 !!! note
```
Comment (shueja) [inline docs/usage/saving.md:36]: Remove this note or change it to something accurate.

File: docs/usage/saving.md [hunk 9]
```diff
@@ -1,23 +1,51 @@
 # Saving
 
+Once trajectories are created by Choreo, they must be saved within your robot project.
+
+Choreo has 2 different kinds of files: a .chor file which stores general configs for your project,
+and multiple .traj files which store individual trajectories. Choreo generates and updates .traj files
+in the same directory(or folder) that your .chor file is stored in.
```
Comment (shueja) [inline docs/usage/saving.md:7]:
```suggestion
in the same directory (or folder) that your .chor file is stored in.
```

File: docs/usage/saving.md [hunk 10]
```diff
@@ -1,23 +1,51 @@
...
+
+## Saving your .chor file
+
+Path following libraries, such as ChoreoLib and PathPlannerLib, require your .traj files to be located
+in a specific directory so that they can be read. For Java and C++ teams, .traj files must be placed under
+the deploy/choreo directory; for python teams, they must be placed in the root directory.
```
Comment (shueja) [inline docs/usage/saving.md:13]:
```suggestion
the `deploy/choreo` directory; for Python teams, they must be placed in the root directory.
```

File: docs/usage/saving.md [hunk 11]
```diff
@@ -1,23 +1,51 @@
...
+
+### For Java and C++ teams:
+
+1. Create a "choreo" folder within the "deploy" folder of your robot project.
```
Comment (shueja) [inline docs/usage/saving.md:17]:
```suggestion
1. Create a `choreo` folder within the `deploy` folder of your robot project.
```

File: docs/usage/saving.md [hunk 12]
```diff
@@ -1,23 +1,51 @@
...
+   The deploy folder can be found under "src/main".
```
Comment (shueja) [inline docs/usage/saving.md:18]:
```suggestion
   The deploy folder can be found under `src/main`.
```

File: docs/usage/saving.md [hunk 13]
```diff
@@ -1,23 +1,51 @@
...
+2. Open the choreo sidebar and click "save project"(see "Sidebar Navigation" for more details).
+3. Navigate to the location of your robot project and find the "src/main/deploy/choreo" folder within it.
```
Comment (shueja) [inline docs/usage/saving.md:20]:
```suggestion
3. Navigate to the location of your robot project and find the `src/main/deploy/choreo` folder within it.
```

**Review by shueja — CHANGES_REQUESTED** [review]
Be more mindful of capitalization and code block usage.

File: docs/usage/saving.md [hunk 14]
```diff
@@ -1,23 +1,51 @@
...
+
 To access the sidebar, click the hamburger icon in the upper left corner.
 
 ![Sidebar image](../media/sidebar.png)
 
 General Info:
 
 !!! note
-    Choreo’s file saving mechanism ties into a WPILib robot root directory, wherein a single `.chor` file lives in the project’s root folder.
+Choreo’s file saving mechanism ties into a WPILib robot root directory, wherein a single `.chor` file lives in the project’s root folder.
```
Comment (shueja) [inline docs/usage/saving.md:37]:
```suggestion

```

File: docs/usage/saving.md [hunk 15]
```diff
@@ -1,23 +1,51 @@
...
+
+## Sidebar Navigation
+
 To access the sidebar, click the hamburger icon in the upper left corner.
 
 ![Sidebar image](../media/sidebar.png)
 
 General Info:
 
 !!! note
```
Comment (shueja) [inline docs/usage/saving.md:36]:
```suggestion

```

**Review by shueja — APPROVED** [review]

**Pull request merged**